networkx/readwrite/graphml.py

```diff
@@ -53,13 +54,17 @@
         pass
 
 @open_file(1,mode='wb')
-def write_graphml(G, path, encoding='utf-8',prettyprint=True):
+def write_graphml(G, path, infer_numeric_types=False, encoding='utf-8',prettyprint=True):
```

Comment:
Should we put the new argument at the end so it is backward compatible for people that call without using keyword?